sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -176,14 +177,15 @@ export default defineComponent({
       formInstance.on("nextPage", prevNextNavigation);
     };
 
-    const setInitialData = (programYearStartDate: string) => {
+    const setInitialData = (programYearStartDate: string, offeringIntensity: OfferingIntensity) => {
```

Comment:
Formatting errors here. 
![image](https://github.com/bcgov/SIMS/assets/54600590/6fb4270b-72c6-4637-9e7f-4216e0c48e0f)

Reply:
VS code extensions and formatting settings are properly configured. Please have a look.